packages/web/server/api/edge-router.ts

```diff
@@ -12,4 +13,5 @@ export const edgeRouter = createTRPCRouter({
   pools: poolsRouter,
   staking: stakingRouter,
   earn: earnRouter,
+  transactions: transactionsRouter,
```

Comment:
Is this PR also applying changes from https://github.com/osmosis-labs/osmosis-frontend/pull/3077

Perhaps this should target one of your other branches instead of stage?